src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -172,6 +177,138 @@ public void dateToCurrentPosition(CaseData caseData) {
         }
     }
 
+
+    public void processHearingsForUpdateRequest(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        // check if update is only on one hearing
+        if(HEARING_NUMBER.equals(caseData.getHearingUpdateFilterType())) {
+            Optional<HearingTypeItem> hearingForUpdate = caseData.getHearingCollection().stream()
+                    .filter(h->h.getValue().getHearingNumber()
+                            .equals(caseData.getSelectedHearingNumberForUpdate()
+                                    .getValue().getCode())).findFirst();
+            if(hearingForUpdate.isPresent()) {
+                if(caseData.getHearingsCollectionForUpdate() == null) {
+
+                    caseData.setHearingsCollectionForUpdate(new ArrayList<>());
+                } else {
+                    caseData.getHearingsCollectionForUpdate().clear();
+                }
```

Comment:
I feel like this can just be simplified to the below. clear and setting it to a new arraylist would do the same thing
```suggestion
                caseData.setHearingsCollectionForUpdate(new ArrayList<>());
```